docs/tutorial/installation.rst

```diff
@@ -23,6 +23,7 @@ Make sure it is available and working::
 
     fedora-messaging --help
 
+If your project uses Docker, include the library in the ``requirements.txt`` file.
```

Comment:
I think it'd be best if in the "install the library and its dependencies" section above we just recommend installing it with pip: "pip install fedora-messaging" with a note below that for installing it directly from the git repository. Something like:
```
Install the library and its dependencies::

    pip install fedora-messaging
    # Alternatively, install it directly from the git repository
    git clone https://github.com/fedora-infra/fedora-messaging.git
    cd fedora-messaging
    pip install -e .
```

Not everyone uses ``requirments.txt`` files so I think the hint about pip-installing is enough to give the reader the package name and they can put it in their requirements file or setup.py or Pipenv file or whatever.

@abompard, do you recall why we recommended installing it from source control rather than PyPI?

Reply:
I don't know, maybe we had no published version at that time.